net/ddns-scripts/Makefile

```diff
@@ -112,6 +112,20 @@ define Package/ddns-scripts_nsupdate/config
 
 endef
 
+###### *************************************************************************
+define Package/ddns-scripts_route53-v1
+    $(call Package/ddns-scripts/Default)
+    TITLE:=Amazon AWS Route 53 API v1
+    DEPENDS:=ddns-scripts +curl +openssl-util +ca-bundle
```

Comment:
Is it good to have a dependency on ca-bundle ?
I personally use ca-certificate.
This dependency possibly force to install a second ca package.

Reply:
you're the expert, so I do whatever you ask me to do. Installing ca-bundle worked for me.

Removed and made this a comment instead